src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -57,6 +58,13 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     logger.info(s"users.ldap-user-id-attr-name = $ldapUserIdAttrName")
     logger.info(s"users.ldap-admin-principal = $ldapAdminPrincipal")
   }
+  val multipartConfig: MultipartConfig = MultipartConfig(
+    location = Option(properties.getString("multipart.location", null)),
```

Comment:
why do so complicated? `getString` already gives a `null` if the property is not set, right? Just wrapping it in `Option` and leaving out the `, null` should be perfectly fine. (not tried it 😉)

```suggestion
    location = Option(properties.getString("multipart.location")),
```

same for the ones below?

Reply:
`new PropertiesConfiguration().getLong("XXX")` throws an exception

Reply:
Of course!!! `"XXX"` is not a `Long`!

Reply:
XXX is the key, expecting to have a long value!!!